site/src/pages/TemplatePage/DAUChart.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { render } from "testHelpers/renderHelpers"
+import { DAUChart, Language } from "./DAUChart"
+
+import { screen } from "@testing-library/react"
+import { ResizeObserver } from "resize-observer"
+
+// The Chart performs dynamic resizes which fail in tests without this.
+Object.defineProperty(window, "ResizeObserver", {
+  value: ResizeObserver,
+})
```

Comment:
If this is needed for these tests, it's usually best to put it in a setup function like `beforeEach` or `beforeAll`. Otherwise it can have unintended side effects.